### PR TITLE
Add sorting to the Incidents list by Incident ID (ascending and descending)

### PR DESCRIPTION
**Pull request by pdcp1**
This closes issue https://github.com/responsible-ai-collaborative/aiid/issues/2425

- Add sorting option by Incident ID on `/summaries/incidents` page
image

### Link to test
https://deploy-preview-2495--staging-aiid.netlify.app/summaries/incidents/

### CODE REVIEW

**Comment by clari182** [comment]
@pdcp1 Looks good! Could you just add the translations for that page? They're missing and since you're modifying it and that page is not too big I think it's worth it.

**Review by clari182 — CHANGES_REQUESTED** [review]
@pdcp1 Looks good! Could you just add the translations for that page? They're missing and since you're modifying it and that page is not too big I think it's worth it.

**Comment by pdcp1** [comment]
> @pdcp1 Looks good! Could you just add the translations for that page? They're missing and since you're modifying it and that page is not too big I think it's worth it.

@clari182 please review. In particular, to the French translations 😉

File: site/gatsby-site/i18n/locales/fr/incidents.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "Incident List": "Liste des Incidents",
+    "This is a simple numeric listing of all incidents and their reports within the database. If you would like to explore the contents of the reports, you should work through the<1> Discover app</1>.": "Il s'agit d'une simple liste numérique de tous les incidents et de leurs rapports dans la base de données. Si vous souhaitez explorer le contenu des rapports, vous devez parcourir les<1> Découvrir les incidents</1>.",
```
Comment (clari182) [inline site/gatsby-site/i18n/locales/fr/incidents.json:3]: Please update to "Il s'agit d'une simple liste numérique de tous les incidents et de leurs rapports dans la base de données. Si vous souhaitez explorer le contenu des rapports, vous devez parcourir <1> Découvrir les incidents</1>."

**Review by clari182 — CHANGES_REQUESTED** [review]
Looks good, just one tiny change and we're good to go!

File: site/gatsby-site/i18n/locales/fr/incidents.json (hunk 1 above)
Comment (pdcp1) [reply site/gatsby-site/i18n/locales/fr/incidents.json]: Done!

**Review by clari182 — APPROVED** [review]

**Pull request merged**